packages/xo-server/src/api/vm.js

```diff
@@ -1159,21 +1159,15 @@ resume.resolve = {
 
 // -------------------------------------------------------------------
 
-export async function revert({ snapshot, snapshotBefore }) {
-  const { id: userId, permission } = this.user
-  await this.checkPermissions(userId, [[snapshot.$snapshot_of, 'operate']])
-  const newSnapshot = await this.getXapi(snapshot).revertVm(
-    snapshot._xapiId,
-    snapshotBefore
-  )
-  if (snapshotBefore && permission !== 'admin') {
-    await this.addAcl(userId, newSnapshot.$id, 'admin')
-  }
+export async function revert({ snapshot }) {
+  await this.checkPermissions(this.user.id, [
+    [snapshot.$snapshot_of, 'operate'],
+  ])
+  await this.getXapi(snapshot).revertVm(snapshot._xapiId)
```

Comment:
I think we should throw if `snapshotBefore` param is true.

Reply:
Yes, it makes sense.

Reply:
IMHO we should still remove it from the `params` list to avoid advertising it via introspection (eg `xo-cli`).

Reply:
Then the first commit alone should do it, right?